modes/segmentation/src/index.tsx

```diff
@@ -156,7 +156,9 @@ function modeFactory({ modeConfiguration }) {
     /** List of extensions that are used by the mode */
     extensions: extensionDependencies,
     /** HangingProtocol used by the mode */
-    hangingProtocol: ['@ohif/mnGrid'],
```

Comment:
This line is restoring what was previously in the segmentation mode - I'm fine not removing this line, as the tests will pass either way now, but I'm just restoring the line to the way it was before the last set of segmentation stuff got added and this line got removed to "fix" the hpscale.